modules/default/weather/providers/smhi.js

```diff
@@ -104,8 +104,12 @@ WeatherProvider.register("smhi", {
 	 * @returns {string} the url for the specified coordinates
 	 */
 	getURL() {
-		let lon = this.config.lon;
-		let lat = this.config.lat;
+		const formatter = new Intl.NumberFormat("en-US", {
+			minimumFractionDigits: 2,
+			maximumFractionDigits: 2
```

Comment:
did not dig in the docs of NumberFormat but would expect `6` instead of `2` as written in the comments above (?)

Reply:
Yeah, i shouldnt code while having a cold :-( Fixed!